transform/mattermost-analytics/analyses/active_user_base.sql

```diff
@@ -74,7 +74,7 @@ rudder_licenses as (
 ), license_data as (
     -- Gather all self-hosted license data from CWS and legacy licenses.
     select
-        license_id
+        license_id.
```

Comment:
intentional?

Reply:
Good catch. Was using this file as a reference and seems this is a miss-type.